lib/preview-frame.ts

```diff
@@ -21,6 +25,24 @@ function loadP5(version: string, cb?: () => void) {
   document.body.appendChild(script);
 }
 
+function loadScripts(urls: string[], cb?: () => void) {
```

Comment:
Yeah, I really like promises, but I'm not actually sure if we include the polyfill in the preview-frame... and I'd like to keep the preview frame as light as possible in terms of external dependencies and stuff, hrm.  Lemme look into it!